src/test/java/fi/otavanopisto/kuntaapi/test/PtvServiceMocker.java

```diff
@@ -0,0 +1,25 @@
+package fi.otavanopisto.kuntaapi.test;
+
+import fi.metatavu.ptv.client.model.V4VmOpenApiService;
+
+@SuppressWarnings ("squid:S1450")
+public class PtvServiceMocker extends AbstractPtvMocker<V4VmOpenApiService> {
+
+  private static final String BASE_PATH = "/ptv/api/v4/Service";
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Refactor your code to get this URI from a customizable parameter. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1075)